### ISSUE TITLE
Fix S6419 FN: Support unsigned right-shift operator (>>>)

### ISSUE THREAD

**Issue by gregory-paidis-sonarsource** [SonarSource/sonar-dotnet#6309]
### Description

FN for S6419 (AzureFunctionsStateless).

Using C#11 unsigned right-shift operator (>>>), the rule is not raising an issue.

### Repro steps

```csharp
[FunctionName("Sample")]
public static void SideEffects()
{
    WithArg(Field >>>= 1);        // FN
}

private static void WithArg(int value) { }
```

### Expected behavior

The rule should raise an issue.

### Actual behavior

The rule does not raise an issue.
